cfgov/cfgov/settings/production.py

```diff
@@ -98,12 +97,16 @@
             'MAX_ENTRIES': 10000,
         },
     },
-    'api_cache':{
+    'api_cache': {
         'BACKEND': 'django.core.cache.backends.locmem.LocMemCache',
         'LOCATION': 'api_cache_memory',
         'TIMEOUT': 3600,
         'OPTIONS': {
             'MAX_ENTRIES': 1000,
         },
+    },
+    'post_preview': {
+        'BACKEND': 'django.core.cache.backends.filebased.FileBasedCache',
+        'LOCATION': '/tmp/post_preview_cache',
```

Comment:
I'm not sure whether this directory needs to be created first or if it will automatically be created.  @rosskarchner ?

Reply:
I don't think it needs to be created automatically-- we aren't doing that for the other cache directories